---
hide:
  - navigation
  - toc
---
# Publications

## Other materials

1. Inno, H., & Peterson, H. (2025). **ELIXIR Estonia: A Year of Growth, Progress, and Impact** (1.0). ELIXIR All Hands 2025 (AHM25), Thessaloniki, Greece. Zenodo. https://doi.org/10.5281/zenodo.15656761
2. Inno, H., & Peterson, H. (2024). **Empowering Life Sciences Research: ELIXIR Estonia** poster. Zenodo. Available from: https://doi.org/10.5281/zenodo.13847963

## Peer reviewed publications

Publications from ELIXIR Europe collaborations are printed in _cursive_.
Tools and services are linked from the title of the corresponding paper.

### 2025

1. Alper, P., D'Anna, F., Droesbeke, B., Andrabi, M., Andrade Buono, R., Bianchini, F., Bösl, K., Chandramouliswaran, I., Cook, M., Faria, D., Fatima, N., Hooft, R., Jareborg, N., Jetten, M., Pilvar, D., Poires-Oliveira, G., Popleteeva, M., Portell-Silva, L., Slifka, J., Suchánek, M., van Gelder, C., Welter, D., Wittig, U., Coppens, F., Goble, C. (2025). **RDMkit: A research data management toolkit for life sciences**. Patterns, 101345. [https://doi.org/10.1016/j.patter.2025.101345](https://doi.org/10.1016/j.patter.2025.101345)
2. Moor, M., Männaste, L., Putrinš, M., Fishman, D., & Kogermann, K. (2025). **FiBar: A tool for analyzing fiber diameters in complex drug delivery systems from scanning electron microscopy images.** European Journal of Pharmaceutical Sciences, 212, 107179. [https://doi.org/10.1016/j.ejps.2025.107179](https://doi.org/10.1016/j.ejps.2025.107179)

### 2024

1. Päll, T., Abroi, A., Avi, R., Niglas, H., Shablinskaja, A., Pauskar, M., Jõgeda, L., Soeorg, H., Kallas, E., Lahesaare, A., Truusalu, K., Hoidmets, D., Sadikova, O., Ratnik, K., Sepp, H., Dotsenko, L., Epštein, J., Suija, H., Kaarna, K., . . .  Huik, K. (2024). **SARS-CoV-2 clade dynamics and their associations with hospitalisations during the first two years of the COVID-19 pandemic.** PLOS ONE, 19(5), e0303176. [https://doi.org/10.1371/journal.pone.0303176](https://doi.org/10.1371/journal.pone.0303176)
2. Pallaseni, A., Peets, E.M., Girling, G. et al. **The interplay of DNA repair context with target sequence predictably biases Cas9-generated mutations.** Nat Commun 15, 10271 (2024). [https://doi.org/10.1038/s41467-024-54566-7](https://doi.org/10.1038/s41467-024-54566-7)
3. Soeorg, H., Abroi, A., Päll, T., Dotsenko, L., Jaaniso, E., Kaarna, K., Lahesaare, A., Naaber, P., Niglas, H., Oopkaup, O. E., Peterson, H., Reisberg, T., Sadikova, O., Smit, S., Talas, U. G., Avi, R., Lutsar, I., & Huik, K. (2024). **Dynamics of SARS-CoV-2 lineages in children and adults in 2021 and 2022.** PLOS ONE, 19(12), e0316213. [https://doi.org/10.1371/journal.pone.0316213](https://doi.org/10.1371/journal.pone.0316213)

### 2023

 1. Kolberg, L., Raudvere, U., Kuzmin, I., Adler, P., Vilo, J., and Peterson, H.
**[g: Profiler—interoperable web service for functional enrichment analysis and gene identifier mapping \(2023 update\) .](https://academic.oup.com/nar/article/51/W1/W207/7152869)**
*Nucleic Acids Research*, 51\(W1\):W207–W212, Jul 2023.
Available from: [https://doi.org/10.1093/nar/gkad347](https://doi.org/10.1093/nar/gkad347)
 2. Kerimov, N., Tambets, R., Hayhurst, J. D., Rahu, I., Kolberg, P., Raudvere, U., Kuzmin, I., Chowdhary, A., Vija, A., Teras, H. J., Kanai, M., Ulirsch, J., Ryten, M., Hardy, J., Guelfi, S., Trabzuni, D., Kim-Hellmuth, S., Rayner, W., Finucane, H., Peterson, H., Mosaku, A., Parkinson, H., and Alasoo, K.
**eQTL Catalogue 2023: New datasets, X chromosome QTLs, and improved detection and visualisation of transcript-level QTLs .**
*PLoS Genetics*, Sep 2023.
Available from: [https://doi.org/10.1371/journal.pgen.1010932](https://doi.org/10.1371/journal.pgen.1010932)
 3. Koeppel, J., Weller, J., Peets, E. M., Pallaseni, A., Kuzmin, I., Raudvere, U., Peterson, H., Liberante, F. G., and Parts, L.
**[Prediction of prime editing insertion efficiencies using sequence features and DNA repair determinants.](https://www.nature.com/articles/s41587-023-01678-y)**
*Nature Biotechnology*, 41:1446–1456, Oct 2023.
Available from: [https://doi.org/10.1038/s41587-023-01678-y](https://doi.org/10.1038/s41587-023-01678-y)

### 2022

 1. Uusküla, A., Kalda, R., Solvak, M., Jürisson, M., Käärik, M., Fischer, K., Keis, A., Raudvere, U., Vilo, J., Peterson, H., Käärik, E., Metspalu, M., Jürgenson, T., Milani, L., Kolberg, L., Tiit, E.-M., and Vassil, K.
**The 1st year of the COVID-19 epidemic in Estonia: a population-based nationwide sequential/consecutive cross-sectional study .**
*Public Health*, 205:150–156, Apr 2022.
Available from: [https://doi.org/10.1016/j.puhe.2022.02.004](https://doi.org/10.1016/j.puhe.2022.02.004)

### 2021

 1. Kerimov, N., Hayhurst, J. D., Peikova, K., Manning, J. R., Walter, P., Kolberg, L., Samoviča, M., Sakthivel, M. P., Kuzmin, I., Trevanion, S. J., Burdett, T., Jupp, S., Parkinson, H., Papatheodorou, I., Yates, A. D., Zerbino, D. R., and Alasoo, K.
**[A compendium of uniformly processed human gene expression and splicing quantitative trait loci.](https://elixir.ut.ee/eqtl/)**
*Nature genetics*, 53:1290–1299, Sep 2021.
Available from: [https://doi.org/10.1038/s41588-021-00924-w](https://doi.org/10.1038/s41588-021-00924-w)

### 2020

 1. Gurwitz, K. T., Gaur, P S., Bellis, L. J., Larcombe, L., Alloza, E., Balint, B. L., Botzki, A., Dimec, J., Angel, V. D., Fernandes, P. L., Korpelainen, E., Krause, R., Kuzak, M., Le Pera, L., Leskošek, B., Lindvall, J. M., Marek, J. D., Martinez, P. A., Muyldermans, T., Nygård, S., Palagi, P. M., Peterson, H., Psomopoulos, F., Spiwok, V., van Gelder, C. W. G., Via, A., Vidak, M., Wibberg, D., Morgan, S. L., and Rustici, G.
***A framework to assess the quality and impact of bioinformatics training across ELIXIR.***
*PLoS Computational Biolology*, Jul 2020.
Available from: [https://doi.org/10.1371/journal.pcbi.1007976](https://doi.org/10.1371/journal.pcbi.1007976)
 2. Fishman, D., Kuzmin, I., Adler, P., Vilo, J., and Peterson, H.
**[PAWER: Protein Array Web ExploreR.](https://biit.cs.ut.ee/pawer/)**
*BMC Bioinformatics*, Sep 2020.
Available from: [https://doi.org/10.1186/s12859-020-03722-z](https://doi.org/10.1186/s12859-020-03722-z)
 3. Ison, J., Ménager, H., Brancotte, B., Jaaniso, E., Salumets, A., Raček, T., Lamprecht, A.-L., Palmblad, M., Kalaš, M., Chmura, P., Hancock, J. M., Schwämmle, V., and Ienasescu, H.-I.
***Community curation of bioinformatics software and data resources.***
*Briefings in Bioinformatics*, 21\(5\):1697–1705, Sep 2020.
Available from: [https://doi.org/10.1093/bib/bbz075](https://doi.org/10.1093/bib/bbz075)
 4. Kolberg, L., Raudvere, U., Kuzmin, I., Vilo, J., and Peterson, H.
**[gprofiler2 – an R package for gene list functional enrichment analysis and namespace conversion toolset g:Profiler \[version 2; peer review: 2 approved\] .](https://cran.r-project.org/web/packages/gprofiler2/index.html)**
*F1000Res*, Nov 2020.
Available from: [https://doi.org/10.12688/f1000research.24956.2](https://doi.org/10.12688/f1000research.24956.2)

### 2019

 1. Raudvere, U., Kolberg, L., Kuzmin, I., Arak, T., Adler, P., Peterson, H., and Vilo, J.
**[g:Profiler: a web server for functional enrichment analysis, conversions of gene lists \(2019 update\) .](https://biit.cs.ut.ee/gprofiler/)**
*Nucleic Acids Research*, 47\(W1\):W191–W198, May 2019.
Available from: [https://doi.org/10.1093/nar/gkz369](https://doi.org/10.1093/nar/gkz369)
 2. Ison, J., Ienasescu, H., Chmura, P., Rydza, E., Ménager, H., Kalaš, M., Schwämmle, V., Grüning, B., Beard, N., Lopez, R., Duvaud, S., Stockinger, H., Persson, B., Vařeková, R. S., Raček, T., Vondrášek, J., Peterson, H., Salumets, A., Jonassen, I., Hooft, R., Nyrönen, T., Valencia, A., Capella, S., Gelpí, J., Zambelli, F., Savakis, B., Leskošek, B., Rapacki, K., Blanchet, C., Jimenez, R., Oliveira, A., Vriend, G., Collin, O., van Helden, J., Løngreen, P., and Brunak, S.
***The bio.tools registry of software tools and data resources for the life sciences .***
*Genome Biology*, Aug 2019.
Available from: [https://doi.org/10.1186/s13059-019-1772-6](https://doi.org/10.1186/s13059-019-1772-6)

### 2018

 1. Kõressaar, T., Lepamets, M., Kaplinski, L., Raime, K., Andreson, R., and Remm, M.
**[Primer3\_masker: integrating masking of template sequence with primer design software .](https://bioinfo.ut.ee/primer3/)**
*Bioinformatics*, 34\(11\):1937–1938, Jun 2018.
Available from: [https://doi.org/10.1093/bioinformatics/bty036](https://doi.org/10.1093/bioinformatics/bty036)
 2. Kolberg, L., Kuzmin, I., Adler, P., Vilo, J., and Peterson, H.
**[funcExplorer: a tool for fast data-driven functional characterisation of high-throughput expression data .](https://biit.cs.ut.ee/funcexplorer/)**
*BMC Genomics volume*, Nov 2018.
Available from: [https://doi.org/10.1186/s12864-018-5176-x](https://doi.org/10.1186/s12864-018-5176-x)

### 2017

 1. Jiménez, R. C., Kuzak, M., Alhamdoosh, M., Barker, M., Batut, B., Borg, M., Capella-Gutierrez, S., Hong, N. C., Cook, M., Corpas, M., Flannery, M., Garcia, L., Gelpí, J. L., Gladman, S., Goble, C., Ferreiro, M. G., Gonzalez-Beltran, A., Griffin, P. C., Grüning, B., Hagberg, J., Holub, P., Hooft, R., Ison, J., Katz, D. S., Leskošek, B., Gómez, F. L., Oliveira, L. J., Mellor, D., Mosbergen, R., Mulder, N., Perez-Riverol, Y., Pergl, R., Pichler, H., Pope, B., Sanz, F., Schneider, M. V., Stodden, V., Suchecki, R., Vařeková, R. S., Talvik, H.-A., Todorov, I., Treloar, A., Tyagi, S., van Gompel, M., Vaughan, D., Via, A., Wang, X., Watson-Haigh, N. S., and Crouch, S.
***Four simple recommendations to encourage best practices in research software \[version 1; peer review: 3 approved\] .***
*F1000Research*, Jun 2017.
Available from: [https://doi.org/10.12688/f1000research.11407.1](https://doi.org/10.12688/f1000research.11407.1)
 2. Hillion, K.-H., Kuzmin, I., Khodak, A., Rasche, E., Crusoe, M., Peterson, H., Ison, J., and Ménager, H.
***Using bio.tools to generate, annotate workbench tool descriptions \[version 1; peer review: 4 approved\] .***
*F1000Research*, Nov 2017.
Available from: [https://doi.org/10.12688/f1000research.12974.1](https://doi.org/10.12688/f1000research.12974.1)
 3. Modhukur, V., Iljasenko, T., Metsalu, T., Lokk, K., Laisk-Podar, T., and Vilo, J.
**[MethSurv: a web tool to perform multivariable survival analysis using DNA methylation data .](https://biit.cs.ut.ee/methsurv/)**
*Epigenomics*, Dec 2017.
Available from: [https://doi.org/10.2217/epi-2017-0118](https://doi.org/10.2217/epi-2017-0118)

### 2016

 1. Ison, J., Rapacki, K., Ménager, H., Kalaš, M., Rydza, E., Chmura, P., Anthon, C., Beard, N., Berka, K., Bolser, D., Booth, T., Bretaudeau, A., Brezovsky, J., Casadio, R., Cesareni, G., Coppens, F., Cornell, M., Cuccuru, G., Davidsen, K., Vedova, G. D., Dogan, T., Doppelt-Azeroual, O., Emery, L., Gasteiger, E., Gatter, T., Goldberg, T., Grosjean, M., Grüning, B., Helmer-Citterich, M., Ienasescu, H., Ioannidis, V., Jespersen, M. C., Jimenez, R., Juty, N., Juvan, P., Koch, M., Laibe, C., Li, J.-W., Licata, L., Mareuil, F., Mičetić, I., Friborg, R. M., Moretti, S., Morris, C., Möller, S., Nenadic, A., Peterson, H., Profiti, G., Rice, P., Romano, P., Roncaglia, P., Saidi, R., Schafferhans, A., Schwämmle, V., Smith, C., Sperotto, M. M., Stockinger, H., Vařeková, R. S., Tosatto, S. C., la Torre, V. d., Uva, P., Via, A., Yachdav, G., Zambelli, F., Vriend, G., Rost, B., Parkinson, H., Løngreen, P., and Brunak, S.
***Tools and data services registry: a community effort to document bioinformatics resources .***
*Nucleic Acids Research*, 44\(D1\):D38–D47, Jan 2016.
Available from: [https://doi.org/10.1093/nar/gkv1116](https://doi.org/10.1093/nar/gkv1116)
 2. Reimand, J., Arak, T., Adler, P., Kolberg, L., Reisberg, S., Peterson, H., and Vilo, J.
**[g:Profiler—a web server for functional interpretation of gene lists \(2016 update\) .](https://biit.cs.ut.ee/gprofiler/page/archives/)**
*Nucleic Acids Research*, 44\(W1\):W83–W89, Apr 2016.
Available from: [https://doi.org/10.1093/nar/gkw199](https://doi.org/10.1093/nar/gkw199)

### 2015

 1. Metsalu, T. and Vilo, J.
**[ClustVis: a web tool for visualizing clustering of multivariate data using Principal Component Analysis and heatmap .](https://biit.cs.ut.ee/clustvis/)**
*Nucleic Acids Research*, 43\(W1\):W566–W570, Jul 2015.
Available from: [https://doi.org/10.1093/nar/gkv468](https://doi.org/10.1093/nar/gkv468)

### Earlier

 1. Untergasser, A., Cutcutache, I., Koressaar, T., Ye, J., Faircloth, B. C., Remm, M., and Rozen, S. G.
**[Primer3—new capabilities and interfaces.](https://bioinfo.ut.ee/primer3/)**
*Nucleic Acids Research*, 40\(15\):e115, Aug 2012.
Available from: [https://doi.org/10.1093/nar/gks596](https://doi.org/10.1093/nar/gks596)
 2. Reimand, J., Arak, T., and Vilo, J.
**[g:Profiler—a web server for functional interpretation of gene lists \(2011 update\) .](https://biit.cs.ut.ee/gprofiler/page/archives/)**
*Nucleic Acids Research*, 39\(suppl\_2\):W307–W315, Jun 2011.
Available from: [https://doi.org/10.1093/nar/gkr378](https://doi.org/10.1093/nar/gkr378)
 3. Adler, P., Kolde, R., Kull, M., Tkachenko, A., Peterson, H., Reimand, J., and Vilo, J.
**[Mining for coexpression across hundreds of datasets using novel rank aggregation and visualization methods .](https://biit.cs.ut.ee/mem/)**
*Genome Biology*, Dec 2009.
Available from: [https://doi.org/10.1186/gb-2009-10-12-r139](https://doi.org/10.1186/gb-2009-10-12-r139)
 4. Adler, P., Reimand, J., Janes, J., Kolde, R., Peterson, H., and Vilo, J.
**[KEGGanim: pathway animations for high-throughput data.](https://biit.cs.ut.ee/kegganim/)**
*Bioinformatics*, 24\(4\):588–590, Feb 2008.
Available from: [https://doi.org/10.1093/bioinformatics/btm581](https://doi.org/10.1093/bioinformatics/btm581)
 5. Reimand, J., Tooming, L., Peterson, H., Adler, P., and Vilo, J.
**[GraphWeb: mining heterogeneous biological networks for gene modules with functional significance .](https://biit.cs.ut.ee/graphweb/)**
*Nucleic Acids Research*, 36\(suppl\_2\):W452–W459, Jul 2008.
Available from: [https://doi.org/10.1093/nar/gkn230](https://doi.org/10.1093/nar/gkn230)
 6. Kaplinski, L. and Remm, M.
**[MultiPLX: Automatic Grouping and Evaluation of PCR Primers.](https://bioinfo.ut.ee/multiplx/)**
*In: Yuryev, A. \(eds\) PCR Primer Design. Methods in Molecular Biology™. Humana Press *, 402:287–303, 2007.
Available from: [https://doi.org/10.1007/978-1-59745-528-2\_14](https://doi.org/10.1007/978-1-59745-528-2_14)
 7. Koressaar, T. and Remm, M.
**[Enhancements and modifications of primer design program Primer3.](https://bioinfo.ut.ee/primer3/)**
*Bioinformatics*, 23\(10\):1289–1291, May 2007.
Available from: [https://doi.org/10.1093/bioinformatics/btm091](https://doi.org/10.1093/bioinformatics/btm091)
 8. Reimand, J., Kull, M., Peterson, H., Hansen, J., and Vilo, J.
**[g:Profiler—a web-based toolset for functional profiling of gene lists from large-scale experiments .](https://biit.cs.ut.ee/gprofiler/page/archives/)**
*Nucleic Acids Research*, 35\(suppl\_2\):W193–W200, Jul 2007.
Available from: [https://doi.org/10.1093/nar/gkm226](https://doi.org/10.1093/nar/gkm226)
 9. Andreson, R., Puurand, T., and Remm, M.
**[SNPmasker: automatic masking of SNPs and repeats across eukaryotic genomes .](https://www.bioinfo.ut.ee/snpmasker/)**
*Nucleic Acids Research*, 34\(suppl\_2\):W651–W655, Jul 2006.
Available from: [https://doi.org/10.1093/nar/gkl125](https://doi.org/10.1093/nar/gkl125)
