---
hide:
  - navigation
  - toc
---
# Services

<style type="text/css">@media screen and (min-width: 769px) {
    .tool-tiles {
      display: flex;
      min-height: 100%;
      align-items: stretch;
    }
  }
</style>

The Estonian ELIXIR node offers various data analysis services and tools for the European and international user community. These include mainly tools, training and bioinformatics infrastructure that have applications across life science and biomedical fields. _All ELIXIR Estonia bioinformatics tools listed below are open and free to use for everyone across the world. We follow [ELIXIR Estonia Services Terms of Use](../terms.md)._

### Services included in the Service Delivery Plan

<div class="tool-tiles">
<tool-tile description="a web tool for visualizing clustering of multivariate data using Principal Component Analysis and heatmap" doi="http://dx.doi.org/10.1093/nar/gkv468" href="https://biit.cs.ut.ee/clustvis/" name="ClustVis" src="/assets/images/tools/ClustVis.png" tags="visualisation, PCA, heatmap, clustering"></tool-tile>
<tool-tile description="a web server for functional enrichment analysis and conversions of gene lists" doi="http://dx.doi.org/10.1093/nar/gkz369" href="https://biit.cs.ut.ee/gprofiler/" name="g:Profiler" src="/assets/images/tools/gp.png" tags=" enrichment analysis, ORA"></tool-tile>
</div>

<div class="tool-tiles">
<tool-tile description="a web tool for mining large biological networks" doi="http://dx.doi.org/10.1093/nar/gkn230" href="https://biit.cs.ut.ee/graphweb/" name="GraphWeb" src="/assets/images/tools/GraphWeb.png" tags="networks"></tool-tile>
<tool-tile description="visualisation of high-throughput data on biological pathway charts" doi="http://dx.doi.org/10.1093/bioinformatics/btm581" href="https://biit.cs.ut.ee/kegganim" name="KEGGanim" src="/assets/images/tools/KEGG.png" tags="visualisation, high-throughput data, pathways"></tool-tile>
<tool-tile description="large-scale query facilities for mining across hundreds of datasets" doi="http://dx.doi.org/10.1186/gb-2009-10-12-r139" href="https://biit.cs.ut.ee/mem" name="MEM" src="/assets/images/tools/MEM.png" tags="rank aggregation, visualisation"></tool-tile>
</div>

<div class="tool-tiles">
<tool-tile description="a tool for grouping PCR primers by various compatibility criteria" doi="http://dx.doi.org/10.1007/978-1-59745-528-2_14" href="https://bioinfo.ut.ee/multiplx" name="MultiPLX" src="/assets/images/tools/MultiPLX.png" tags="PCR"></tool-tile>
<tool-tile description="a program for designing PCR primers and oligos" doi="http://dx.doi.org/10.1093/nar/gks596" href="https://bioinfo.ut.ee/primer3" name="Primer3" src="/assets/images/tools/Primer3.png" tags="PCR"></tool-tile>
<tool-tile description="automatic masking of SNPs and repeats across eukaryotic genomes" doi="http://dx.doi.org/10.1093/nar/gkl125" href="https://bioinfo.ut.ee/snpmasker" name="SNPmasker" src="/assets/images/tools/SNPmasker.png" tags="SNP"></tool-tile>
</div>

### Emerging services

<div class="tool-tiles">
<tool-tile description="extracts the most significant biological features (GO annotations, pathways, etc.) of a dataset" doi="http://dx.doi.org/10.1186/s12864-018-5176-x" href="https://biit.cs.ut.ee/funcexplorer" name="funcExplorer" src="/assets/images/tools/funcExplorer.png" tags="clustering"></tool-tile>
<tool-tile description="a web tool to perform multivariable survival analysis using DNA methylation data" doi="http://dx.doi.org/10.2217/epi-2017-0118" href="https://biit.cs.ut.ee/methsurv" name="MethSurv" src="/assets/images/tools/MethSurv.png" tags="methylation,"></tool-tile>
<tool-tile description="a software tool for analysing protein microarray data" doi="http://dx.doi.org/10.1186/s12859-020-03722-z" href="https://biit.cs.ut.ee/pawer" name="PAWER" src="/assets/images/tools/PAWER.png" tags="protein microarray"></tool-tile>
</div>

### In collaboration with <a href="https://kauralasoo.github.io">Alasoo Group</a>

<div class="tool-tiles" style="width: 50%;">
<tool-tile description="" doi="https://www.nature.com/articles/s41588-021-00924-w" href="https://elixir.ut.ee/eqtl/" name="eQTL Catalogue Browser" src="/assets/images/tools/eqtl.png" tags="gene expression, splicing, QTL"></tool-tile>
</div>

### In collaboration with <a href="https://www.sanger.ac.uk/group/parts-group/">Parts Group</a> from <a href="https://www.sanger.ac.uk">Wellcome Sanger Institute</a>

<div class="tool-tiles">
<tool-tile description="prediction of prime editing insertion efficiencies using sequence features and DNA repair determinants" doi="https://www.nature.com/articles/s41587-023-01678-y" href="https://elixir.ut.ee/minsepie/" name="MinsePIE" src="/assets/images/tools/minsepie.png" tags="CRISPR/Cas9, prime editing"></tool-tile>
<tool-tile description="a tool for predicting base editing outcomes using position-specific sequence determinants" doi="https://academic.oup.com/nar/article/50/6/3551/6548303" href="https://elixir.ut.ee/forecast-be/" name="FORECasT-BE" src="/assets/images/tools/forecast-be.png" tags="CRISPR/Cas9, base editing"></tool-tile>
</div>
<div class="tool-tiles">
<tool-tile description="a tool for predicting the distribution of mutations at a CRISPR/Cas9 cut" doi="https://www.biorxiv.org/content/10.1101/2023.06.28.546891v1" href="https://elixir.ut.ee/forecast-repair/" name="FORECasT-repair" src="/assets/images/tools/forecast-repair.png" tags="CRISPR/Cas9"></tool-tile>
<tool-tile description="a tool for predicting the mutations generated by repair of Cas9-induced double-strand breaks" doi="https://www.nature.com/articles/nbt.4317" href="https://elixir.ut.ee/forecast/" name="FORECasT" src="/assets/images/tools/forecast.png" tags="CRISPR/Cas9"></tool-tile>
</div>
