---
draft: false
date: 2024-01-29
authors:
  - Diana
categories:
  - workshop
  - bioinformatics
  - enrichment-analysis
hide:
  - toc
---

# Online course: Enrichment Analysis

Experiments designed to quantify gene expression often yield hundreds of genes that show statistically significant differences between groups of interest. Once differentially expressed genes are identified, enrichment analysis (EA) methods can be used to explore the biological functions associated with these genes. EA methods allow us to identify groups of genes (e.g. particular pathways) that are over-represented, thereby offering insights into biological mechanisms. One of the EA methods frequently used for high-throughput gene expression data analysis is Gene Set Enrichment Analysis (GSEA). This course will cover GSEA and alternative enrichment methods. Because the implementation of GSEA is directly linked to databases that annotate the function of genes in a cell, the course will also give an overview of functional annotation databases such as Gene Ontology.

Application deadline: 26 February 2024

[More Info and Registration](https://www.sib.swiss/training/course/20240311_ENRIC) 

<!-- more -->

## General information 

* __Audience:__ Biologists who are eager to perform functional annotation of a set of differentially expressed genes.
* __Time:__ 11 March 2024
* __Location:__ Online
* __Fees:__ Registration fees are 100 CHF for academics and 500 CHF for for-profit companies.

## Prerequisites

* Statistics, beginner level (T-test, multiple testing methods).
* R, beginner level (Rstudio, how to install a library, matrix and data frame manipulation, import and export data from text files).

## Learning objectives

At the end of the course, participants will be able to:

* Distinguish available enrichment analysis methods.
* Apply GSEA and over-representation analysis using R.
* Determine whether the genes of a GO term have a statistically significant difference in expression or not.
* Learn where to find other gene sets in databases (e.g. KEGG, oncogenic gene sets) and use them in R.

[Registration form](https://www.sib.swiss/training/course-apply/20240311_ENRIC)

[More Info and Registration](https://www.sib.swiss/training/course/20240311_ENRIC) 