---
draft: false
date: 2024-06-05
authors:
  - Diana
categories:
  - webinar
  - ISMARA
  - CREMA
hide:
  - toc
---

# Webinar: ISMARA and CREMA: automatically inferring gene regulatory networks from gene expression, chromatin accessibility or epigenome data

This course is aimed at PhD students, post-docs and researchers in life sciences wanting to make computational inferences about gene regulation from gene expression data, chromatin accessibility and epigenetic mark data. The course is targeted to a wide audience ranging from computational biologists and bioinformatics researchers who regularly analyse transcriptome and epigenome data, to experimental researchers interested in inferring key regulatory interactions in their own data.

Application deadline: 28 August 2024

Date:  30 August 2024

[More Info and Registration](https://www.sib.swiss/training/course/20240830_ISMCM) 

<!-- more -->
## General information 

* __Registration deadline:__ 28 August 2024
* __Time:__ 30 August 2024
* __Location:__ Online
* __Fees:__ Free
* __Level:__ Beginner 

## Course description

Motif Activity Response Analysis (MARA) is a powerful method that automatically infers the most important regulatory interactions acting in a given biological system of interest from either gene expression or chromatin state data. It takes as input either gene expression (i.e. RNA-seq or micro-array), chromatin accessibility (i.e. ATAC-seq or DNase-seq) or epigenome (i.e. ChIP-seq of histone modifications), typically in the form of raw sequencing data files, and then models these data in terms of computationally predicted transcription factor binding sites. In this way, the method infers the key regulators acting in the system of interest, how they change their activity across the input samples, the genomic target regions and genes of each regulator, and so on. MARA analysis has been completely automated and is provided through two online tools: ISMARA and CREMA. Researcher can simply upload their raw data and obtain a comprehensive report on the regulatory network structure within an interactive graphical web interface. In this 1-day course, participants will learn how to use the online tools with their own data, including all the functionalities that the interface provides, how to explore the results that the system produces, how to perform tailored downstream analyses, and how to interpret the various results and predictions.


## Learning objectives

At the end of the course, the participants are expected to:

* Describe the analysis approach used by ISMARA and CREMA.
* Use the ISMARA and CREMA web server.
* Use the interactive web interface to generate tailored analyses.
* Gain a detailed understanding of the meaning and interpretation of the results and predictions provided by these methods.

## Prerequisites

* Participants are expected to be familiar with the molecular biology of gene regulation in higher eukaryotes. No specific bioinformatics skills are necessary.
* To optimally benefit from the course, participants are highly encouraged to upload their own datasets at least a week in advance. This will allow them to explore the results of these analyses on their own data.

## Registration

[Registration form](https://www.sib.swiss/training/course-apply/20240830_ISMCM)

[More Info](https://www.sib.swiss/training/course/20240830_ISMCM) 