---
draft: false
date: 2024-08-09
authors:
  - Diana
categories:
  - Bioinformatics
  - NGS
  - Next generation sequencing
  - Genome Variant Analysis

hide:
  - toc
---

# Online: NGS - Genome Variant Analysis

This course is intended for life scientists who are already familiar with general concepts of NGS technologies and want to expand their knowledge and skills on variant analysis. Course material is available for free.

Application deadline: 15 August 2024

Date: 05 - 06 September 2024

[More Info](https://www.sib.swiss/training/course/20240905_NGSGV) 

<!-- more -->

## General information 

* __Location:__  Online
* __Application deadline:__ 15 August 2024
* __Date:__ 05 - 06 September 2024
* __Level:__ Beginner
* __Fees:__
  * Academic: 200 CHF
  * For-profit: 1000 CHF
* __Materials:__ https://sib-swiss.github.io/NGS-variants-training/2024.2/

## Description

The detection of genetic variation is of major interest in various disciplines spanning from ecology and evolution research to inherited disease discovery and precision oncology. Next generation sequencing (NGS) methods are very powerful for the detection of genomic variants. Thanks to its throughput and cost-efficiency it enables the detection of a large number of variants in a large number of samples. In this two-day course we will cover the steps from read alignment to variant calling and annotation. We will mainly focus on the detection of germline mutations by following the GATK best practices. The training materials for this course are in its dedicated GitHub page.

## Prerequisites

Participants should have knowledge in NGS techniques, quality control and alignment to a reference genome. Participants should have a basic understanding of working with command line tools on Unix-based systems. Participants should have their own computers.

## Learning outcomes

At the end of the course participants should be able to:

* Understand important aspects of NGS and read alignment for variant analysis
* Perform a read alignment ready for variant analysis
* Perform variant calling according to GATK best practices
* Perform a variant annotation

[More Info](https://www.sib.swiss/training/course/20240905_NGSGV) 

