---
draft: false
date: 2024-10-17
authors:
  - Diana
categories:
  - EBI
  - Bioinformatics
  - RNA-seq
  - Python
  - single-cell
  
hide:
  - toc
---

# Virtual course by EMBL-EBI: Single-cell RNA-seq analysis with Python

This course is aimed at wet-lab researchers who are generating, planning on generating, or working with single cell RNA sequencing data.

Registration deadline: 17 November 2024

Date: 	17 - 21 February 2025

[More Info](https://www.ebi.ac.uk/training/events/single-cell-rna-seq-analysis-with-python-2025/)

<!-- more -->

## Description 

This course covers the analysis of single cell RNA sequencing (scRNA-seq) data using Python and command line tools. Participants will be guided through droplet-based scRNA-seq analysis pipelines from raw reads to cell clusters. You will explore and interpret single-cell RNA seq data using Python as well as the Single Cell Expression Atlas. Finally, you will put their knowledge into practice through a group challenge on the last day.

Please note that you will not analyse your own data as part of the course. There will, however, be ample opportunity to discuss your research and ideas with other course participants and trainers.

Participants will learn via a mix of pre-recorded lectures, live presentations, and trainer Q&A sessions. Practical experience will be developed through group activities and trainer-led computational exercises. Live sessions will be delivered using Zoom with additional support and asynchronous communication via Slack.

Pre-recorded material may be provided before the course starts that participants will need to watch, read or work through to gain the most out of the actual training event. In the week before the course, there will be a brief induction session. Computational practicals will run on EMBL-EBI's virtual training infrastructure, meaning you will not require access to a powerful computer or install complex software on your own machines.

Participants will need to be available between the hours of 09:00 â€“ 17:30 BST each day of the course. Trainers will be available to assist, answer questions, and provide further explanations during these times.

Please note that we will operate this course virtually. Hybrid options are not currently available. We reserve the right to change the format of this course or cancel it.

## Prerequisites

Some experience with Python is beneficial. During the course, some of the practicals will use a Linux-based command line interface. We recommend all successul applicants acquire/brush up on their basic skills in Python and the command line before attending the course. There are many tutorials available online and here are some that may be of help:

* To complete the following suggested tutorials you may want to install Ubuntu for Windows Users if you are using a computer with a Windows Operating System.
* Basic introduction to the Unix environment: https://swcarpentry.github.io/shell-novice/
* Introduction to programming in Python: http://swcarpentry.github.io/python-novice-gapminder/ 

## Learning outcomes 

* Explain the steps in the scRNA-seq pipeline
* Repeat the course analysis of scRNA-seq data from extraction to cluster maps
* Recognise decision-making steps along the analysis pipeline and justify your decisions, from experimental design to final visualisation
* Employ appropriate data standards for repository submission and contribution to global cell atlases

[More Info](https://www.ebi.ac.uk/training/events/single-cell-rna-seq-analysis-with-python-2025/)