---
draft: false
date: 2024-04-09
authors:
  - Diana
categories:
  - workshop
  - Python
  - single-cell
  - single-cell-transcriptomics
hide:
  - toc
---

# Course at Bern, Switzerland: Single-Cell Transcriptomics with Python

This course is addressed to life scientists and bioinformaticians familiar with some next-generation sequencing approaches and plan to use scRNA-seq in their research projects, and who are interested in learning how to navigate and analyze such data.

Registration deadline: 13 May 2024

[More Info and Registration](https://www.sib.swiss/training/course/20240527_SCTPY) 

<!-- more -->

## General information 

* __Time:__ 27 - 29 May 2024
* __Registration deadline:__ 13 May 2024
* __Location:__  University of Bern
* __Fees:__ 
    *  Academic: 300 CHF
    *  For-profit: 1500 CHF

## Description

Single-cell RNA sequencing (scRNA-seq) can measure the gene expression of complex biological systems at the level of individual cells, enabling scientists to generate detailed tissue atlases describing the transcriptomic profiles of thousands or even millions of cells. While scRNA-seq has become a popular technique in diverse fields of biological research, the required expertise for handling such datasets has restricted its use among the larger scientific community. The aim of this 3-day course is to empower researchers to start applying the fundamental scRNA-seq analysis pipeline to their own data. We will outline how to design and interpret results of a scRNA-seq dataset and, in teams, explore the basics of preprocessing and analysis in Python on real data. We will discuss common concerns in the field, including preprocessing choices, dimensionality reduction, cell type clustering and identification, batch effect correction, and pseudotime methods. By the end of the course, participants will be able to run CellRanger, evaluate the quality of a scRNA-seq experiment, perform scanpy analysis on their own data, and confidently communicate about how to overcome potential bottlenecks. The course will be taught in Python.

## Learning outcomes

At the end of the course, the participants are expected to:

* distinguish advantages and pitfalls of scRNA-seq
* design their own scRNA-seq experiment
* apply a downstream analysis using Python
* interpret and overcome common technical challenges of single-cell analysis

## Prerequisites

Participants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques, or have already followed the "NGS - Quality control, Alignment, Visualisation". Knowledge in RNA sequencing is mandatory. Basic knowledge of Python is required. 
You are required to bring your own laptop. Data and tutorials used in the course will be provided ahead of the start date and will be executed using Jupyter notebooks and Anaconda.

## Registration

[Registration](https://www.sib.swiss/training/course-apply/20240527_SCTPY) 

[More Info](https://www.sib.swiss/training/course/20240527_SCTPY) 

