---
draft: false
date: 2025-01-16
authors:
  - Diana
categories:
  - single cell
  - single-cell-transcriptomics
  - RNA-seq

hide:
  - toc
---

# Course in Italy: scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis

The workshop is suitable for life scientists who are new to single cell gene expression technology data analysis. It is open to PhD and Post-Doc research scientists, as well as team leaders and PIs. This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package. 

Deadline for applications: 20th January 2025

Course date: 10-12 February 2025

[More Info](https://elixir-iib-training.github.io/site/2025-02-10-scRNA_seq_Workshop) 

<!-- more -->

## Description

This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package https://github.com/kendomaniac/rCASC. Participants will delve into experimental design principles, quality control for single-cell sequence outputs, and data reduction and clustering techniques to identify cell subpopulations. The course also offers an overview of the computational theory behind scRNA-seq tools, empowering participants to make informed decisions during analysis. Practical, hands-on sessions will allow attendees to work with real datasets and integrate results to extract meaningful biological insights.
Venue: University of Bari

The course includes a fee of 150 Euros for academic attendees and 400 Euros for industry professionals. Participants are expected to pay their own travel and accommodation costs (if any).

## Requirements

No prior knowledge of statistics or computing skills is required. However, a basic understanding of R programming is preferred. Prior knowledge of single-cell sequencing technologies is required.

## Learning outcomes

* Describe the importance of experimental design to ask sensible biological questions at single cell level.
* Assess the quality of your data.
* Describe limits and strength of clustering in scRNA-seq.
* Identify genes driving cluster formation in scRNA-seq.
* Annotate cell types
* Performe batch correction and experiment integration on scRNA-seq data.


[More Info and registration](https://elixir-iib-training.github.io/site/2025-02-10-scRNA_seq_Workshop) 

